FoodSuggestions/app/src/main/res/layout/activity_main.xml

```diff
@@ -1,21 +1,67 @@
 <?xml version="1.0" encoding="utf-8"?>
-<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
```

Comment:
Please remove attributes related to ConstraintLayout.

Reply:
Fixed